Nos/Views/Discover/FeaturedAuthor+Cohort2.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+extension FeaturedAuthor {
+    static let cohort2 = [
+        FeaturedAuthor(
+            name: "Lou",
+            npub: "npub1rr3678k7ajms2sht0cqqeawy86sdd5ahn6akfj8zex9ng82zuh0sz8nywd",
+            categories: [.gaming]
+        ),
+        FeaturedAuthor(
+            name: "We Distribute",
+            npub: "npub1w9wuqc3s6lr25c4sgj52werj3tngvt43qrccqrher4wvn7tjm32s2ck403",
+            categories: [.news, .tech]
+        ),
+        FeaturedAuthor(
+            name: "Simon Howard",
+            npub: "npub1rt5h26ukmqsqa29ggt0h98tq2skgqr85f3znhwxule4z4pjhhd3qzs5k94",
+            categories: [.music]
+        ),
+        FeaturedAuthor(
+            name: "BTCPhoto",
+            npub: "npub1vjl6n2llukcc6pe3am2hkwqh8twzh2ymlp7pdrdfq5tlqg08y26sd7ygzx",
+            categories: [.art]
+        ),
+        FeaturedAuthor(
+            name: "Sam Hain",
+            npub: "npub1df47g7a39usamq83aula72zdz23fx9xw5rrfmd0v6p9t20n5u0ss2eqez9",
+            categories: [.art]
+        ),
+        FeaturedAuthor(
+            name: "Joanna",
+            npub: "npub1a2a85jwde32zjsjk02ujasydqc3t2w9rfgfe97amm0r4d9mepfxsxf3fnn",
+            categories: [.activists, .health]
+        ),
+        FeaturedAuthor(
+            name: "Josh Brown",
+            npub: "npub1yl8jc6znttslcpj3p6p8vuq98awu6w0xh4lqtu0lkjr772kpx4ysfqvz34",
+            categories: [.art]
```

Comment:
Maybe we need to run this by Linda, but I always thought I should be in Photography rather than Art. Now that we have a Photography category, we can actually do that.

```suggestion
            categories: [.photography]
```

Reply:
I think I can just change it to photography.